sympy/core/sympify.py

```diff
@@ -51,6 +51,41 @@ class CantSympify(object):
     pass
 
 
+def convert_numpy_types(a):
+    """
+    Converts a numpy datatype input to an appropriate sympy type.
+
+    Examples
+    ========
+
+    >>> from sympy import Float
+    >>> from numpy import int64, float128, float32
+
+    >>> Float(np.int64(5))
+    5.00000000000000
+
+    >>> Float(np.float128(5))
```

Comment:
Is float128 available on all platforms? If not, we should remove this from the doctest so it doesn't fail.

Reply:
And also I'm not sure even if it is always available if it's always the same precision.

Reply:
Okay, for now I'll just remove it.

Reply:
Can also make it conditional on the platform (see below) but probably easier to remove altogether.
import sys
is_64bits = sys.maxsize > 2**32